### DISCUSSION TITLE
Welcome to Zettlr Discussions!

### DISCUSSION THREAD

**Discussion by nathanlesage** [Zettlr/Zettlr#2320] [Announcements]
> **Update Dec 26, 2024**: We have now re-opened the Zettlr Community Forum and would like to invite you over there, rather than here! Join the discussion at https://forum.zettlr.com/!

## Welcome!

I have just activated the Discussions feature here on the Zettlr repository out of two reasons. First, more and more people have a Github account and being active here even as a non-developer is more and more normalized. This means that even people who do not code themselves have Github accounts, and for those people I think the discussions here are perfect.

The second reason is that I wanted to deactivate the old forum on my server since it involves quite the amount of maintaining which I cannot really guarantee. The more stuff I can outsource towards third parties the more time I have to manage the actual development of the app. And I'm pretty positive that all of you would agree that the app should be the main concern given my limited time with my PhD!

So basically, this Discussions-section serves the same purpose as the old forum, which'll be made read-only in some time.

What you can therefore do here is the same as on the forum:

* Discuss new feature ideas or how to enhance an existing feature _before_ actually opening the corresponding issue (the benefit of Github Discussions is that I can comfortably create an issue out of a discussion, so this is great!)
* Ask for help/guidance
* Share workflows and ideas

Feel free to (re)-introduce yourselves below to the community!

## Other Places for the Zettlr Community

If you're new to Zettlr, the community is at several places at once! Feel free to join wherever you like:

* [Reddit](https://www.reddit.com/r/Zettlr) — similar to this forum here, but allows cross-posting from other discussions and works slightly different
* [Discord](https://discord.gg/PcfS3DM9Xj) — for all the real-time communication (chatting, talking, quickly discussion stuff that doesn't merit its own full-fledged discussion here, etc.)

**Comment by break6700** [comment]
Hello Internet! I go by Break on most platforms. I'm a recently graduated R&D engineer and have been using Zettlr for around 2 years. I love FOSS and warm hugs!

**Comment by KenzoDrumasaki** [comment]
Hello, I'm a doctoral student in art history. I've been using Zettlr for about a year, off and on, to write my dissertation. It's really inspiring to see Zettlr make such progress!

**Comment by leo-hstone** [comment]
I appreciate the usage of Github Discussions as I hate to create more and more accounts. The old forum is still activly linked to on several places. Unfortunatly, I am not experienced enough to fix them by myself, so I am leaving a note here :) 

- CONTRIBUTING.md
- "New Issues" template screen

**Comment by suchoudh** [comment]
hi folks 

from India. Wonderful work. 
Hope there are more contributors from India and work on making Indian languages contribution to Zettlr. 

thanks 
Sunil

**Comment by sanoopnp** [comment]
Is there an option to open a specific file from workspace by searching the filename

**Comment by cutervino** [comment]
For non-tech guys like me, being new to Zettlr (Which I find much better than other apps) it is hard for me to even share my ideas. I am writing my first book, and reading the Zettlr tutorials, sometimes, is frustrating. Here's an example. “At the core of Zettlr, there is a mighty engine that is capable of untangling what you write and extract common sections such as pages (`p.` and `pp.`), chapters (`chapter`) and sections (`sec.` or `§`), even in multiple languages!” Frankly, I'm lost. It doesn't instruct me how to actually do it. My suggestion: Please write the tutorials thinking on guys like me, start from ground zero when explaining anything, in a manner that it'll be impossible not to understand you. Finally, at the end of the tutorial: “If you have improvement suggestions, we are always happy to hear you out!”. So I tried to comment on Github, and I could find where to insert my suggestions.

**Reply by cutervino** [reply]
Sorry, my last sentence should be: 'and I could NOT find where to insert my suggestions'.
